internal/api/v1/user/contacts/contact_filter_query_builder.go

```diff
@@ -29,5 +29,13 @@ func (c *ContactFilterQueryBuilder) Build() (url.Values, error) {
 	params.AddPair("paymail", c.ContactFilter.Paymail)
 	params.AddPair("pubKey", c.ContactFilter.PubKey)
 	params.AddPair("status", c.ContactFilter.Status)
+	return params, nil
+}
+
+func (c *ContactFilterQueryBuilder) Build() (url.Values, error) {
+	params, err := c.BuildExtendedURLValues()
```

Comment:
same as in above, can params be `nil` ?

Reply:
This question is already addressed in the previous comment.